.github/workflows/checks.yml

```diff
@@ -12,9 +12,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: gradle/wrapper-validation-action@v1
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v3
         with:
-          java-version: 11
+          distribution: 'zulu'
+          java-version: 17
```

Comment:
Can you move java and library updates to a separate PR?

Reply:
Yep!

Reply:
https://github.com/Laimiux/lce/pull/44